### PR TITLE
[TASK] Remove outdated note for Fluid

### PR DESCRIPTION
**Pull request by sypets**

### CODE REVIEW

**Comment by sypets** [comment]
Note is now removed!

File: Documentation/ApiOverview/Fluid/Index.rst [hunk 1]
```diff
@@ -32,9 +32,9 @@ You can use Fluid in TYPO3 to do one of the following:
 
 .. note::
 
-   This page was created in 2020 to serve as a start page for Fluid in TYPO3.
-   It contains a :ref:`fluid-introduction` as a subpage. Further pages will
-   be added later.
+   This page was created in 2020 to serve as a start page for Fluid in TYPO3
+   (independently of Extbase). Further pages are being migrated from the "Extbase
```
Comment (mbrodala) [inline Documentation/ApiOverview/Fluid/Index.rst:36]:
```suggestion
   independent from Extbase. Further pages are being migrated from the "Extbase
```

Comment (mbrodala) [reply Documentation/ApiOverview/Fluid/Index.rst]: ;-)

**Review by mbrodala — APPROVED** [review]

**Comment by sypets** [comment]
Will backport to 11.5 manually.

**Pull request merged**